diff --git a/lib/main/electron/TestRunner.js b/lib/main/electron/TestRunner.js
index 44858627c971995e6d308eea1d63348b84ac12e1..12e0042eb8a03ff8ef79a678b9e8f9e92e78d99c 100644
--- a/lib/main/electron/TestRunner.js
+++ b/lib/main/electron/TestRunner.js
@@ -73,7 +73,9 @@ async function startWorker(rootDir, target, config) {
                     // Ignored
                 }
             }
-            child.kill("SIGKILL");
+            try {
+                child.kill("SIGKILL");
+            } catch {}
         });
         return child;
     });
diff --git a/lib/main/rpc/RPCProcess.js b/lib/main/rpc/RPCProcess.js
index 19ffdd59a36612e64c517c9832fbd656d2a961b8..f839b35dcc3cbc4b4307dfcd42472722c17f5dc9 100644
--- a/lib/main/rpc/RPCProcess.js
+++ b/lib/main/rpc/RPCProcess.js
@@ -74,7 +74,9 @@ class RPCProcess {
                 // Ignored
             }
         }
-        this.subProcess?.kill("SIGKILL");
+        try {
+            this.subProcess?.kill("SIGKILL");
+        } catch {}
         delete this.server;
         this.isAlive = false;
     }
