diff --git a/node_modules/@kayahr/jest-electron-runner/lib/main/electron/TestRunner.js b/node_modules/@kayahr/jest-electron-runner/lib/main/electron/TestRunner.js
index 4485862..12e0042 100644
--- a/node_modules/@kayahr/jest-electron-runner/lib/main/electron/TestRunner.js
+++ b/node_modules/@kayahr/jest-electron-runner/lib/main/electron/TestRunner.js
@@ -73,7 +73,9 @@ async function startWorker(rootDir, target, config) {
                     // Ignored
                 }
             }
-            child.kill("SIGKILL");
+            try {
+                child.kill("SIGKILL");
+            } catch {}
         });
         return child;
     });
diff --git a/node_modules/@kayahr/jest-electron-runner/lib/main/rpc/RPCProcess.js b/node_modules/@kayahr/jest-electron-runner/lib/main/rpc/RPCProcess.js
index 19ffdd5..bd9f53b 100644
--- a/node_modules/@kayahr/jest-electron-runner/lib/main/rpc/RPCProcess.js
+++ b/node_modules/@kayahr/jest-electron-runner/lib/main/rpc/RPCProcess.js
@@ -74,7 +74,9 @@ class RPCProcess {
                 // Ignored
             }
         }
-        this.subProcess?.kill("SIGKILL");
+        try {
+          this.subProcess?.kill("SIGKILL");
+        } catch {}
         delete this.server;
         this.isAlive = false;
     }
